lib/index.coffee

```diff
@@ -35,6 +35,13 @@ garbageCollect = (reclaimSpace) ->
 		# Request deletion of each image
 		console.log("Removing image: #{image.repoTags[0]}")
 		docker.getImage(image.id).removeAsync()
+		.catch (e) ->
+			# TODO: If an image fails to be removed, this means that the total space
```

Comment:
Before we returned a rejected promise, which would make it possible for callers to know that the requested space to reclaim hadn't been recovered, we could at least return `false` this case and below do
```coffee
.then (results) ->
	console.log('Done.')
	return _.every(results)
```
which will return true/false depending on whether we freed up the expected space, except for the case where the requested reclaim space is bigger than is possible, which would require
```coffee
.then (results) ->
	console.log('Done.')
	return size >= reclaimSpace and _.every(results)
```
- but I think in the case we've reclaimed the max possible it's still ok to return true (eg the first case)